src/aosm/azext_aosm/tests/latest/recordings/test_vnf_nsd_publish_and_delete.yaml

```diff
@@ -0,0 +1,8240 @@
+interactions:
```

Comment:
YAML accepts comments - does the test work ok if you add a comment block to the top of this file saying what it is?

Reply:
Oh no, this is autogenerated. Ignore that.